autoload/lsp/omni.vim

```diff
@@ -340,8 +347,9 @@ function! lsp#omni#get_vim_completion_items(options) abort
             let l:start_characters += [l:default_start_character]
         endif
 
+        let l:expandable = get(l:completion_item, 'insertTextFormat', 1) == 2
         if l:expandable
-            let l:vim_complete_item['word'] = lsp#utils#make_valid_word(substitute(l:vim_complete_item['word'], '\$[0-9]\+\|\${\%(\\.\|[^}]\)\+}', '', 'g'))
+            let l:vim_complete_item['word'] = lsp#utils#make_valid_word(l:vim_complete_item['word'])
```

Comment:
Remove `substitute(var, "'\$[0-9]\+\|\${\%(\\.\|[^}]\)\+}'", 'g')` because it is duplicated to 
https://github.com/prabirshrestha/vim-lsp/blob/c0482f0fb407c29a88875616175784a886f131c3/autoload/lsp/utils.vim#L450